README.MARCJOSEPH.md

```diff
@@ -0,0 +1,32 @@
+# Car Rules 
+general Usage Rules
+
+1.	The vehicle can only go to church, school, and restaurants. All other destinations require permission.
+2. Passengers are limited to Mom and/or Dad. No other passengers (including siblings) are currently allowed. Exceptions require permission for each occurrence.
+3.	The car must be kept clean, no trash or food will be present upon exiting the vehicle.
+
+
+## Gas Maintenance and Fueling
+maintenance considerations
+
+1. Gas will be provided only for trips to and from school,church, work or job interviews; any personal outings are your financial responsibility. For personal outings not related to schooling or the above, fuel usage must be calculated in advance and paid for by you. Parents will contribute $100 for maintenance every two months and $60 for fuel weekly, the car may not be driven unless it meets maintenance standards. The car should never be brought home with less than 1/4 of a tank. Insurance for the car is your responsibility.
+### Scheduling 	
+blurb
+
+1.	Your schedule must be edited and tuned for the day before using the car.
+## Revocation requirements 
+1.	Failure to follow the rules may result in revocation of vehicle usage privileges.
+2.	Vehicle usage privileges may be revoked based on a three-strike system:
+  - The first strike will result in the removal of personal outings, such as visiting friends.
```

Comment:
vehicle privileges will be revoked based on the severity or repeat offenses. For repeat offenses we will use a 3 strike system (first strike removes car access for 2 days, second strike 2 weeks, third strike 1 month, 4th strike total revocation).

Reply:
we can reset strikes every 6 months. Strikes must be listed in the "STRIKELOG.MARCJOSEPH.md" to count